Rakefile

```diff
@@ -43,15 +43,23 @@ task :restore_quick do
   system 'tools/paket.exe', 'restore', clr_command: true
 end
 
+task :restore_dotnetcli do
+  system "dotnet", %W|restore Expecto.netcore/Expecto.netcore.fsproj|
+end
+
 desc 'restore all nugets as per the packages.config files'
-task :restore => [:paket_bootstrap, :restore_quick, :paket_files]
+task :restore => [:paket_bootstrap, :restore_quick, :paket_files, :restore_dotnetcli]
 
 desc 'Perform full build'
-build :compile => [:versioning, :restore, :assembly_info] do |b|
+build :compile => [:versioning, :restore, :assembly_info, :build_dotnetcli] do |b|
   b.prop 'Configuration', Configuration
   b.sln = 'Expecto.Tests/Expecto.Tests.fsproj'
 end
 
+task :build_dotnetcli => [:versioning, :restore_dotnetcli, :assembly_info] do
+  system "dotnet", %W|build -c #{Configuration} -f netstandard1.6 Expecto.netcore/Expecto.netcore.fsproj|
```

Comment:
You may want to add `-v n` argument to `dotnet commands. At least in CI, if there is already an option to know when build is CI or not.
For example for `dotnet build -v n` this will write compiler arguments and usually this is really good for debug errors from log